packages/reports/tests/integration/components/dropdown-parameter-picker-test.ts

```diff
@@ -0,0 +1,84 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, findAll, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { TestContext as Context } from 'ember-test-helpers';
+// @ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import FragmentFactory from 'navi-core/services/fragment-factory';
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+//@ts-ignore
+import { selectChoose } from 'ember-power-select/test-support';
+
+interface TestContext extends Context {
+  filter: FilterFragment;
+  parameterIndex: number;
+  updateParameters: (key: string, value: string) => void;
+  parameterKeys: string[];
+}
+
+module('Integration | Component | dropdown parameter picker', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function (this: TestContext) {
+    const factory: FragmentFactory = this.owner.lookup('service:fragment-factory');
+    this.set('parameterIndex', 0);
+    this.set('parameterKeys', ['grain']);
+    this.set('selectedParameter', 'day');
```

Comment:
```suggestion
```

Reply:
Did you mean to delete the parameterKeys line?

Reply:
yes :)